### ISSUE TITLE
Use image alt text as header text as fallback

### ISSUE THREAD

**Issue by yusf** [codimd/react-client#450]
I have the following scenario:
```
# ![Header](imgsrc)

## Header

### Header
```

Instead of text, I want an image at the top of my document. This screws up the ToC but it doesn't have to if the image is used as a header _and_ if the image comes with an alt text. 

I propose that the app falls back to using the alt text of an image if available.

**Comment by mrdrogdrog** [comment]
This issue is no longer relevant, because it's already solved in the new react frontend.

**Comment by DerMolly** [comment]
This need some tweaking here https://github.com/codimd/react-client/blob/73007ef59774d1acb811f4ac2690d3574f2afc60/src/components/editor/editor.tsx#L51

**Comment by mrdrogdrog** [comment]
Sry, I misread the issue text.
But still we can't solve this issue here. The TOC is generated by https://github.com/nagaozen/markdown-it-toc-done-right. You can open an issue in that project.
